ops/jenkins/bfd-cbc-build/Dockerfile

```diff
@@ -0,0 +1,35 @@
+ARG VERSION_JAVA
+ARG MAVEN_VERSION
+
+FROM maven:${MAVEN_VERSION}-openjdk-${VERSION_JAVA} AS base
+
+# NOTE: necessary duplicate entry
+ARG VERSION_JAVA
```

Comment:
any reason for `VERSION_JAVA` instead of `JAVA_VERSION`?

Reply:
> any reason for `VERSION_JAVA` instead of `JAVA_VERSION`?

I might have been doing something real dumb, but... I ran into an instance where a local `$JAVA_VERSION`, e.g. `11.0.14.1` was coming through instead of the shorter, more innocent value of `11`. I took the lazy way out and didn't give it much more thought, and renamed the variable!